File: SwingApps/apps/cc/applets/zombicide/AWTZombicideBoardBuilder.java
package cc.applets.zombicide;

import cc.lib.board.CustomBoard;
import cc.lib.swing.AWTBoardBuilder;

public class AWTZombicideBoardBuilder extends AWTBoardBuilder {

    @Override
    protected CustomBoard newBoard() {
        return null;
    }

    @Override
    protected String getPropertiesFileName() {
        return "zombicide.bb.properties";
    }

    @Override
    protected String getDefaultBoardFileName() {
        return "zombicide.backup.board";
    }
}

File: SwingApps/apps/cc/applets/zombicide/AWTZombicideBoardBuilder.kt
package cc.applets.zombicide

import cc.lib.board.BCell
import cc.lib.board.BEdge
import cc.lib.board.BVertex
import cc.lib.board.CustomBoard
import cc.lib.swing.AWTBoardBuilder

class AWTZombicideBoardBuilder : AWTBoardBuilder<BVertex, BEdge, BCell, CustomBoard<BVertex, BEdge, BCell>>(
	"zombicide.bb.properties", "zombicide.backup.board"
) {
	override fun newBoard(): CustomBoard<BVertex, BEdge, BCell> {
		TODO()
	}
}